src/pages/profile/profile-sections/student-training-section.js

```diff
@@ -2,71 +2,70 @@ import TextInput from "../../../components/form/textInput";
-          />
-          <TextInput
-            label="Specialism*"
-            type="text"
-            name="specialism"
-            value={profile.specialism || ""}
-            readOnly={!isTeacher}
-            icon={<LockIcon />}
-            onChange={handleInput}
-          />
-          <TextInput
-            label="Cohort*"
-            type="text"
-            name="cohort"
-            value={`Cohort ${profile.cohortId}` || ""}
-            readOnly={!isTeacher}
-            icon={<LockIcon />}
-            onChange={handleInput}
-          />
-          <TextInput
-            label="Start Date*"
-            type="text"
-            name="start-date"
-            value={
-              profile.startDate
-                ? new Date(profile.startDate).toLocaleDateString()
-                : ""
-            }
-            readOnly={true}
-            icon={<LockIcon />}
-          />
-          <TextInput
-            label="End Date*"
-            type="text"
-            name="end-date"
-            value={
-              profile.endDate
-                ? new Date(profile.endDate).toLocaleDateString()
-                : ""
-            }
-            readOnly={true}
-            icon={<LockIcon />}
-          />
-        </div>
+  const isTeacher = user.role === "TEACHER";
+  return (
+    <div className="training-info">
+      <h3>Training info</h3>
+      <div className="training profile-form-inputs">
+        <TextInput
+          label="Role*"
+          type="text"
+          name="role"
+          value={
+            profile.role
+              ? `${profile.role[0].toUpperCase()}${profile.role
+                  .slice(1)
+                  .toLowerCase()}`
+              : ""
+          }
+          readOnly={!isTeacher}
+          icon={<LockIcon />}
+          onChange={handleInput}
+        />
+        <TextInput
+          label="Specialism*"
+          type="text"
+          name="specialism"
+          value={profile.specialism || ""}
+          readOnly={!isTeacher}
+          icon={<LockIcon />}
+          onChange={handleInput}
+        />
+        <TextInput
+          label="Cohort*"
+          type="text"
+          name="cohort"
+          value={profile.cohortId ? `Cohort ${profile.cohortId}` : ""}
+          readOnly={!isTeacher}
+          icon={<LockIcon />}
+          onChange={handleInput}
+        />
+        <TextInput
+          label="Start Date*"
+          type="text"
+          name="start-date"
+          value={
+            profile.startDate
+              ? new Date(profile.startDate).toLocaleDateString()
+              : ""
+          }
+          readOnly={true}
+          icon={<LockIcon />}
+        />
+        <TextInput
+          label="End Date*"
+          type="text"
+          name="end-date"
+          value={
+            profile.endDate
+              ? new Date(profile.endDate).toLocaleDateString()
+              : ""
+          }
+          readOnly={true}
```

Comment:
Minor but when a prop is `true` you can just have `readOnly` which would be evaluated as true